crates/bevy_winit/src/lib.rs

```diff
@@ -719,3 +719,12 @@ fn handle_create_window_events(
         }
     }
 }
+
+/// Map [`bevy_window::CursorGrabMode`] to [`winit::window::CursorGrabMode`].
+fn winit_grab_mode(mode: bevy_window::CursorGrabMode) -> winit::window::CursorGrabMode {
```

Comment:
could you put this with its friends in https://github.com/bevyengine/bevy/blob/main/crates/bevy_winit/src/converters.rs

Reply:
Sure, didn't know this before :smile:

Reply:
Done.